### PR TITLE
Laboratory work #3, Bushueva Nataliya - 21FPL2

### PR DESCRIPTION
**Pull request by NataliaBush**

### CODE REVIEW

File: lab_3_keywords_textrank/main.py [hunk 1]
```diff
@@ -51,7 +52,11 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        clean_text = []
+        for symbol in text:
+            if symbol not in self._punctuation:
+                clean_text += symbol
+        return tuple(" ".join(clean_text).lower().split())
```
Comment (artyomtugaryov) [inline lab_3_keywords_textrank/main.py:59]: better to save `" ".join(clean_text).lower().split()` to a variable and then cast to tuple - difficult to read

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

**Comment by artyomtugaryov** [comment]
there are not new updates :(

File: lab_3_keywords_textrank/main.py [hunk 2]
```diff
@@ -51,7 +53,12 @@ def _clean_and_tokenize(self, text: str) -> tuple[str, ...]:
             tuple[str, ...]
                 clean lowercase tokens
         """
-        pass
+        clean_text = ""
+        for symbol in text.lower():
+            if symbol not in self._punctuation:
+                clean_text += symbol
+        tokens = tuple(clean_text.split())
```
Comment (artyomtugaryov) [inline lab_3_keywords_textrank/main.py:60]: do not create a variable. Return it immediately

File: lab_3_keywords_textrank/main.py [hunk 3]
```diff
@@ -66,9 +73,14 @@ def _remove_stop_words(self, tokens: tuple[str, ...]) -> tuple[str, ...]:
             tuple[str, ...]
                 tokens without stop-words
         """
-        pass
+        words = []
+        for token in tokens:
```
Comment (artyomtugaryov) [inline lab_3_keywords_textrank/main.py:77]: again - do it in one line

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

File: lab_3_keywords_textrank/main.py (hunk 2 above)
Comment (NataliaBush) [reply lab_3_keywords_textrank/main.py:60]: done

File: lab_3_keywords_textrank/main.py (hunk 3 above)
Comment (NataliaBush) [reply lab_3_keywords_textrank/main.py]: done

**Pull request closed without merging**